bower.json

```diff
@@ -36,9 +36,10 @@
     "angular-chart.js": "1.1.1",
     "components-font-awesome": "4.7.0",
     "angular-ui-select": "0.17.1",
-    "JsBarcode": "jsbarcode#3.8.0",
-    "angular-touch": "1.6.6",
-    "angular-growl-notifications": "2.5.0"
+    "JsBarcode": "jsbarcode#3.5.9",
+    "angular-touch": "1.6.5",
```

Comment:
This angular-touch version should match the other angular versions.  We are currently at `1.6.6`.